README.md

```diff
@@ -373,6 +373,13 @@ after a colon, e.g. `5C:-54`.
 dablin_gtk -d ~/bin/dab2eti -c 11D -C 5C,7B,11A,11C,11D -s 0xd911
 ```
 
+You also can use an Airspy or Airspy mini, but you have to specify the
```

Comment:
It is indeed an issue that the auto gain doesn't work for all versions `eti-cmdline`. Hence I added a new parameter `-G` in 978400cd8f8e2fbd45428e65368a642a3af1a017 to allow the use of the respective default gain instead. So also e.g. the HackRF version will work now. So I'm afraid (or rather happy, in the end) that this workaround is not needed anymore.

Reply:
Ok, corrected in later commit 9e90615 and added examples for eti-cmdline-rawfiles.